pkg/apis/config/defaults.go

```diff
@@ -21,6 +21,8 @@ import (
 	"errors"
 	"fmt"
 
+	eventingv1 "knative.dev/eventing/pkg/apis/duck/v1"
```

Comment:
tiny nit, for consistency would you mind changing this import:
eventingv1 => eventingduckv1?